src/users/dto/update-user.dto.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  IsDate,
+  IsEmail,
+  IsNotEmpty,
+  IsOptional,
+  IsString,
+  MaxLength,
+  MinLength,
+} from 'class-validator';
+import { Type } from 'class-transformer';
+
+export class UpdateUserDto {
```

Comment:
the same as CreateUserDto, maybe to union them

Reply:
Fixed